rdm/checklists/62304_2006-05_class_c.txt

```diff
@@ -0,0 +1,3 @@
+# Audit checklist for IEC62304 version 2006-05 Class C products
```

Comment:
Note that we will probably have to update `setup.py` to include the checklists files in the pypi distribution.